bip-tombriar-compressed-transactions.mediawiki

```diff
@@ -0,0 +1,293 @@
+<pre>
+  BIP: ?
+  Layer: API/RPC
+  Title: Compressed Transactions RPC
+  Author: Tom Briar <tombriar11@protonmail.com>
+  Comments-URI: https://github.com/bitcoin/bitcoin/pull/29134
+                https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-August/021924.html
```

Comment:
I think this is mixing up the "Comments-URI" and the "Post-History" headers.
```suggestion
  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-tombriar-compressed-transactions
```

Reply:
Fixed, Thanks!